### ISSUE TITLE
Sometimes "Resource Loading Error" toast right after booting up modpack.

### ISSUE THREAD

**Issue by Gbergz** [KubeJS-Mods/KubeJS#456]
### Minecraft Version

1.18.2

### KubeJS Version

1802.5.4-build.533

### Rhino Version

1802.1.14-build.225

### Architectury Version

4.9.83-fabric

### Forge/Fabric Version

Fabric

### Describe your issue

Sometimes booting up my modpack I get a "resource loading error" that pops up top right corner right after getting to main menu.

Looking at the logs it looks like it has something to do with KubeJS, wondering if anyone has any idea on why this happens? 
As it only happens sometimes, kind of rare but at the same time not.

Also sometimes when this does happen, some custom items texture go missing, also the name localization goes with it aswell, for all items. <- This is happening without that error in the logs aswell. 

### Crash report/logs

https://gist.github.com/Gbergz/eecd10ec0c18a2408c89fd6978155aef

**Comment by ChiefArug** [comment]
Can you reroduce this with only KubeJS and deps? If not, this is probably a conflict with another mod that does dynamic assets/resource packs or similar to KubeJS.

Due to the nature of the error its very hard to determine what the other mod is, so you would have to do a binary search (although given the error doesn't happen all the time that wouldn't be easy).

**Comment by Gbergz** [comment]
Yeah I've actually already tried to do a binary search, it's very hard when the issue doesn't happen every launch.. :S

**Comment by MaxNeedsSnacks** [comment]
We unfortunately don't have the time to support Minecraft versions older than 1.20 anymore, so I'll be closing this for now. You might still be able to find help on our Discord if you submit a ticket with the exact errors, and if it turns out this is a major KubeJS bug, we will of course go ahead and fix it still.
